smart-send-logistics/includes/frontend/class-ss-shipping-frontend.php

```diff
@@ -20,9 +26,12 @@ class SS_Shipping_Frontend
         /**
          * Init and hook in the integration.
          */
+        public $carries = '';
+
         public function __construct()
         {
             $this->init_hooks();
+            $this->include_endpoint_class();
```

Comment:
Why do you need to call this function? I mean it creates a new instance of the `SS_Shipping_Api_Endpoint` class, but it is not used for anything or stored anywhere?

Reply:
The include_endpoint_class method instantiates the SS_Shipping_Api_Endpoint class, passing the current instance ($this). This enables the SS_Shipping_Api_Endpoint class to utilize methods and properties of the current class, allowing it to properly register and handle REST API endpoints for fetching pickup points and shipping carrier information.